config/_default/params.toml

```diff
@@ -158,6 +158,7 @@ link_authors = true
 # Load JS plugins
 #   E.g. To load `/assets/js/custom.js`, set `plugins_js = ["custom"]`.
 plugins_js  = []
+plugins_css = ["custom"]
```

Comment:
I was wrong, you can remove this now, it is only needed for older versions of Hugo